sdks/parsers/src/main/java/org/gbif/pipelines/parsers/parsers/location/parser/LocationMatcher.java

```diff
@@ -131,6 +142,24 @@ public LocationMatcher additionalTransform(UnaryOperator<LatLng> transformation)
     return Optional.empty();
   }
 
+  private Optional<GadmFeatures> getGadmFromCoordinates(LatLng latLng) {
+    if (latLng.isValid()) {
+      GeocodeResponse geocodeResponse = null;
+      try {
+        geocodeResponse = geocodeKvStore.get(latLng);
+      } catch (NoSuchElementException | NullPointerException ex) {
```

Comment:
catching those exceptions here looks like there's bug somewhere else

Reply:
It's not required. I'd copied the call from the method above (the original geocoding) without checking.

Removing here and above.